wire-library/wire-runtime/src/jvmMain/kotlin/com/squareup/wire/internal/FieldBinding.kt

```diff
@@ -142,4 +181,83 @@ class FieldBinding<M : Message<M, B>, B : Message.Builder<M, B>> internal constr
   operator fun get(message: M): Any? = messageField.get(message)
 
   internal fun getFromBuilder(builder: B): Any? = builderField.get(builder)
+
+  fun jsonStringAdapter(syntax: Syntax): JsonFormatter<*>? {
+    return when (adapterString) {
+      "com.squareup.wire.ProtoAdapter#BYTES" -> ByteStringJsonFormatter
+      "com.squareup.wire.ProtoAdapter#INT64",
+      "com.squareup.wire.ProtoAdapter#SFIXED64",
+      "com.squareup.wire.ProtoAdapter#SINT64" -> LongAsStringJsonFormatter
+      "com.squareup.wire.ProtoAdapter#FIXED64" -> UnsignedLongAsStringJsonFormatter
```

Comment:
Do we wanna use them for proto2 as well? I was thinking we'd add that for proto3 only.

Reply:
Good call.

Reply:
That's was probably why the tests were failing. It's green now!